### PR TITLE
docs :: querying :: functions :: label_replace

### PR DESCRIPTION
**Pull request by SmedbergM**
Clarify documentation for label_replace() because of ambiguities between label keys and label values.

### CODE REVIEW

**Comment by SmedbergM** [comment]
@roidelapluie I don't see a separate maintainer for Prometheus docs listed, can you route this to whom it needs to go?

**Comment by roidelapluie** [comment]
Thank you, let's gather @hdost & @RichiH's opinions

File: docs/querying/functions.md [hunk 1]
```diff
@@ -272,24 +272,22 @@ For each timeseries in `v`, `label_join(v instant-vector, dst_label string, sepa
 using `separator` and returns the timeseries with the label `dst_label` containing the joined value.
 There can be any number of `src_labels` in this function.
 
-This example will return a vector with each time series having a `foo` label with the value `a,b,c` added to it:
+This example will return a vector with each time series having a `foo` label added to it, with the value `a,b,c`:
```
Comment (RichiH) [inline docs/querying/functions.md:275]: This does not look like an improvement to me; rest LGTM

**Review by RichiH — CHANGES_REQUESTED** [review]

File: docs/querying/functions.md (hunk 1 above)
Comment (SmedbergM) [reply docs/querying/functions.md:275]: I think I was objecting to the pronoun-antecedent mismatch ("it" refers to "time series" not "label"), but the whole sentence is awkward. Maybe this:
> This example will transform a vector of time series, by adding to each a label `foo` with value `a,b,c`

?

Comment (RichiH) [reply docs/querying/functions.md:275]: It's the label which has the value added, so it's grammatically correct.

Comment (SmedbergM) [reply docs/querying/functions.md]: I've reverted the line in question.

**Review by RichiH — APPROVED** [review]

**Comment by RichiH** [comment]
Thanks!

**Pull request merged**